exercism.go

```diff
@@ -85,6 +85,9 @@ func (e *Exercism) SubmitComment(comment []byte, uuid string) error {
 	}
 
 	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
```

Comment:
Oh, thanks. Yeah, my bad. I must not have run the linter against this. Ironically...